plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -302,4 +432,25 @@ private void fixLocation(ExtendedSymbolInformation symbol) {
       fixLocation(child);
     }
   }
+
+  private List<ClasspathEntryDto> convertToClasspathEntryDto(
```

Comment:
We really should think about extending the dto generator to generate pojo->dto conversion. But outside of the scope of this PR.